examples/advanced/edge/jobs/pt_job.py

```diff
@@ -150,20 +152,24 @@ def main():
         default=10,
         help="Number of global federated rounds under sync mode, total globalmodel version under async mode will multiply this number by the total number of devices",
     )
-    parser.add_argument(
-        "--output_dir",
-        type=str,
-        default="/tmp/nvflare/workspaces/edge_example/prod_00/admin@nvidia.com/transfer",
-        help="Output directory for the recipe",
-    )
+    parser.add_argument("--workspace_dir", type=str, default="/tmp/nvflare/workspaces", help="Workspace directory")
     parser.add_argument(
         "--no_delay",
         action="store_true",
         help="If set, disable communication delay and device speed variations (set to 0.0)",
     )
+    parser.add_argument(
+        "--export_job",
+        action="store_true",
+        help="If set, export the recipe to the admin'stransfer directory",
+    )
+    parser.add_argument("--project_name", type=str, default="edge_example", help="Project name")
 
     args = parser.parse_args()
 
+    prod_dir = os.path.join(args.workspace_dir, args.project_name, "prod_00")
```

Comment:
note that this is only true if the user does provision for 1 time and use that,
or next time it will be prod_01, prod_02

or if they delete the old provisioned folders next time before start.